dap-etl/transformation/src/test/scala/org/broadinstitute/monster/dap/common/TransformationHelperSpec.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.monster.dap.common
+
+import com.spotify.scio.testing.PipelineSpec
+import org.apache.beam.sdk.options.{PipelineOptions, PipelineOptionsFactory}
+import org.scalatest.matchers.should.Matchers
+
+class TransformationHelperSpec extends PipelineSpec with Matchers {
```

Comment:
Actually, I think the spec is failing.

Reply:
Ah, forgot to add the needed fixture data file.